# detection message for each object in scene
string obj_class
geometry_msgs/Point point